dexter/README.md

```diff
@@ -3,10 +3,10 @@
 Dexter is a logging middleware inspired by [expressjs/morgan](https://github.com/expressjs/morgan). It is configurable and can be used throughout the request-resource-response lifecycle.
 
 ```typescript
-import { Drash } from "https://deno.land/x/drash@v1.x/mod.ts";
+import { Drash } from "https://deno.land/x/drash@v{latest}/mod.ts";
 
 // Import the Dexter middleware function
-import { Dexter } from "https://deno.land/x/drash_middleware@v0.x/dexter/mod.ts";
+import { Dexter } from "https://deno.land/x/drash_middleware@v{latest}/dexter/mod.ts";
```

Comment:
Should we not be explicit about the middleware repo version?

Reply:
let me think about this

Reply:
ok. yea. we should.

Reply:
53b3ad0